Content.Shared/Movement/Components/MovementSpeedModifierComponent.cs

```diff
@@ -63,26 +63,26 @@ private float _baseSprintSpeedVV
         /// The negative velocity applied for friction when weightless and providing inputs.
         /// </summary>
         [ViewVariables(VVAccess.ReadWrite), DataField]
-        public float WeightlessFriction = DefaultWeightlessFriction;
+        public float WeightlessFriction { get; set; } = DefaultWeightlessFriction;
```

Comment:
sloth meant make one function per variable, not adding the auto properties to the fields.

Reply:
actually just realized the comp doesn't get dirtied this way nevermind my question